### PR TITLE
CYB-79: File Tailer integration tests

### PR DESCRIPTION
**Pull request by ghost**
File Tailer integration tests

### CODE REVIEW

File: it/file-tailer-integration-tests/pom.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright © 2014 Cask Data, Inc.
```
Comment (awholegunch) [inline it/file-tailer-integration-tests/pom.xml]: Remove leading "~"

File: it/file-tailer-integration-tests/src/test/java/co/cask/cdap/filetailer/FileTailerIT.java [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+ */
+public class FileTailerIT {
+
+  public static final String CONFIG_NAME = "fileTailerITConfig";
+
+  private static final int ENTRY_NUMBER = 1;
+  private static final int WRITING_INTERVAL = 1000;
+  private static final int SLEEP_TIME = 5000;
+  private static final String LOG_MESSAGE = "165.225.156.91 - - [09/Jan/2014:21:28:53 -0400]" +
+    " \"GET /index.html HTTP/1.1\" 200 225 \"http://continuuity.com\" \"Mozilla/4.08 [en] (Win98; I ;Nav)\"";
```
Comment (awholegunch) [inline it/file-tailer-integration-tests/src/test/java/co/cask/cdap/filetailer/FileTailerIT.java]: http://example.com ?

File: it/file-tailer-integration-tests/src/test/resources/logback.xml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  the License.
+  -->
+
+<!--                                                                         -->
+<!-- Stream Client Java API Example Configuration                            -->
+<!--                                                                         -->
+<!-- We use the LogBack project for logging in the Stream Client Java API    -->
+<!-- Example.                                                                -->
+<!-- The manual for Logback can be found here: http://logback.qos.ch/manual  -->
+<!--                                                                         -->
```
Comment (awholegunch) [inline it/file-tailer-integration-tests/src/test/resources/logback.xml]: This comment block should be written as per the above:

**Comment by awholegunch** [comment]
Docs LGTM.

**Comment by sreevatsanraman** [comment]
lgtm

**Pull request merged**